news/ca/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN">
+<html>
+<head>
+<link rel="icon" href="/favicon.ico" type="image/x-icon">
+<link rel="shortcut icon" href="/favicon.ico" type="image/x-icon">
+<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
+<title>GnuCash - empty directory</title>
```

Comment:
@jralls , @gjanssens Do you have any idea, if and where this file is used?
If yes: Do we really want iso-8859-1 here and in the other news/*/index files and what is it's purpose? E.g. de has none. Aside of that having files text here

Reply:
It just puts up a link back to the main page. I guess it's there in case someone tries to go to http://www.gnucash.org/news directly; a bit nicer than the usual access denied error.

The charset attribute tells the browser what encoding the file uses. If the file contains only ASCII (as is the case with news/index.html) then it doesn't matter since all charsets handle ASCII. If there are non-ascii characters (as is the case for both German and Catalan) then it's better to have a charset attribute and of course to actually use that charset to prepare the file. My general preference is to always use UTF-8.